internal/api/provider/oidc.go

```diff
@@ -50,7 +50,11 @@ func ParseIDToken(ctx context.Context, provider *oidc.Provider, config *oidc.Con
 	case IssuerLinkedin:
 		token, data, err = parseLinkedinIDToken(token)
 	default:
-		token, data, err = parseGenericIDToken(token)
+		if IsAzureIssuer(token.Issuer) {
```

Comment:
can we make this exist in it's own `case` block?

Reply:
Nope, the Azure issuer is dynamic and not a fixed string.